src/components/Tool/FixSendNewsletter/Contact.tsx

```diff
@@ -177,16 +162,23 @@ const Contact = ({
                       style={{ height: '100%' }}
                       p={2}
                     >
-                      <Avatar
-                        src=""
-                        style={{
-                          width: theme.spacing(7),
-                          height: theme.spacing(7),
-                        }}
-                      />
+                      {primaryPerson.firstName && (
+                        <Avatar
+                          src=""
+                          style={{
+                            width: theme.spacing(7),
+                            height: theme.spacing(7),
+                          }}
+                        />
+                      )}
                       <Box display="flex" flexDirection="column" ml={2}>
-                        <Typography variant="h6">{name || ''}</Typography>
-                        <Typography>{email || ''}</Typography>
+                        <Typography variant="h6">
+                          {`${primaryPerson.firstName} ${primaryPerson.lastName}` ||
+                            ''}
```

Comment:
I don't think the `|| ''` here does anything.... When both the first name and last names are empty, `' '` is a truthy value so it will be returned instead of `''`.